Resources/config/request_body_param_converter.xml

```diff
@@ -4,6 +4,10 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="fos_rest.converter.request_body.class">FOS\RestBundle\Request\RequestBodyParamConverter</parameter>
```

Comment:
I suggest removing all ``*.class`` parameters in the new major version as this practice is discouraged in Symfony (we need a major version to stop using parameters, but we should not wait until the next one)